src/main/java/com/catcher/core/service/ScheduleService.java

```diff
@@ -0,0 +1,98 @@
+package com.catcher.core.service;
+
+import com.catcher.common.exception.BaseException;
+import com.catcher.common.exception.BaseResponseStatus;
+import com.catcher.core.database.ScheduleDetailRepository;
+import com.catcher.core.database.ScheduleRepository;
+import com.catcher.core.database.UserItemRepository;
+import com.catcher.core.domain.entity.*;
+import com.catcher.core.domain.entity.enums.ItemType;
+import com.catcher.core.domain.entity.enums.ScheduleStatus;
+import com.catcher.core.dto.request.ScheduleDetailRequest;
+import com.catcher.core.dto.response.TempScheduleResponse;
+import com.catcher.core.port.CatcherItemPort;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RequiredArgsConstructor
+@Service
+public class ScheduleService {
+    private final ScheduleRepository scheduleRepository;
+    private final UserItemRepository userItemRepository;
+    private final CatcherItemPort catcherItemPort;
+    private final ScheduleDetailRepository scheduleDetailRepository;
+
```

Comment:
```suggestion
@Transactional
```
조회도 트랜잭션에 포함되니 조회도 붙여주세요!

Reply:
(DB Replication 상황이거나 별도의 상황에서는 @Transcation(readonly=true)를 붙여야 할것 같은데, 위 상황은 단순 조회 코드인 것 같은데, 트랜잭션을 붙여야할까요?

Reply:
@Transcation(readonly=true) 요고 붙이는거 깜박했습니다! 👍